charts/service-library/templates/tests/_pod.yaml

```
@@ -0,0 +1,25 @@
+{{- define "service-library.tests.pod" -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ .Release.Name }}-acceptance-tests
+  labels: {{- include "service-library.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
```

Auto cleanup test pods:
```suggestion
    "helm.sh/hook": test
    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
```